client/src/parent/chores/ChoreCard.tsx

```diff
@@ -22,16 +27,19 @@ type Prop = {
 
 const useStyles = makeStyles((theme: Theme) =>
   createStyles({
-    description: {
-      textAlign: "left",
-    },
     cardContent: {
       paddingTop: "0px",
       paddingBottom: "8px",
     },
+    avatar: {
+      backgroundColor: theme.palette.primary.main,
+    },
   })
 );
 
+const yesterday = new Date(Date.now());
```

Comment:
@marZeu8 Datehandling in JavaScript scheint etwas umständlich zu sein. Hier hab ich mir den gestrigen Tag für die Validierung zusammengestellt.

Reply:
Vielleicht könnten wir auch mit dem eine einfache Formatierung vornehmen.. :)
https://docs.microsoft.com/de-de/dotnet/api/system.windows.forms.datetimepicker.mindate?view=net-5.0